File: Python/Equipo.py
#!/usr/bin/python
# -*- coding: UTF-8 -*-
import ScrumMaster
import Integrante
from typing import List

class Equipo(object):
	def getNombre(self) -> String:
		return self.__nombre

	def setNombre(self, aNombre : String) -> None:
		self.__nombre = aNombre

	def getId(self) -> int:
		return self.__id

	def setId(self, aId : int) -> None:
		self.__id = aId

	def addIntegrate(self, aIntegrante : Integrante):
		pass

	def toString(self):
		pass

	def getScrumMaster(self) -> ScrumMaster:
		return self.scrumMaster

	def setScrumMaster(self, aScrumMaster : ScrumMaster) -> None:
		self.scrumMaster = aScrumMaster

	def Equipo(self, aId : int, aNombre : String, aSm : ScrumMaster):
		pass

	def __init__(self):
		self.__nombre : String = None
		self.__id : int = None
		self.__scrumMaster : ScrumMaster = None
		self.integrantes = []
		"""# @AssociationMultiplicity *
		# @AssociationKind Aggregation"""

File: Java/Equipo.java

package scrum;

import java.util.Vector;

public class Equipo {

    private String nombre;
    private int id;
    private ScrumMaster scrumMaster;
    public Vector<Integrante> integrantes = new Vector<Integrante>();

    public Equipo(String nombre, int id, ScrumMaster scrumMaster) {
        this.nombre = nombre;
        this.id = id;
        this.scrumMaster = scrumMaster;
    }

    
    
    public String getNombre() {
        return this.nombre;
    }

    public void setNombre(String aNombre) {
        this.nombre = aNombre;
    }

    public int getId() {
        return this.id;
    }

    public void setId(int aId) {
        this.id = aId;
    }

    public void addIntegrate(Integrante aIntegrante) {
        throw new UnsupportedOperationException();
    }


    public ScrumMaster getScrumMaster() {
        return this.scrumMaster;
    }

    public void setScrumMaster(ScrumMaster aScrumMaster) {
        this.scrumMaster = aScrumMaster;
    }

    @Override
    public String toString() {
        StringBuilder sb = new StringBuilder();
        sb.append("Equipo{");
        sb.append("nombre=").append(nombre);
        sb.append(", id=").append(id);
        sb.append(", scrumMaster=").append(scrumMaster);
        sb.append(", integrantes=").append(integrantes);
        sb.append('}');
        return sb.toString();
    }

    
}
